src/app.js

```diff
@@ -1,21 +1,27 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  // eslint-disable-next-line no-console
+
+  const { url } = req;
+  const [text, queryString] = url.substring(1).split('?');
+  const params = new URLSearchParams(queryString);
+
+  res.setHeader('Content-Type', 'application/json');
+
+  res.end(JSON.stringify({
+    parts: text.split('/'),
+    query: Object.fromEntries(params.entries()),
+  }));
+});
+
+server.listen(PORT, () => {
+  // eslint-disable-next-line no-console
+  console.log(`Server is running on http://localhost:${PORT}`);
+});
+
+http.get(`http://localhost:${PORT}/hello/world/123?x=1&search=some`);
```

Comment:
How it's related to the server? It seems like a code you used to test the server. You can create separate files for testing or remove the code before submitting PR.